openai/openai.go

```diff
@@ -55,6 +55,32 @@ func (oac *OpenAiClient) Init() {
 	oac.client = openai.NewClient(openaiToken)
 }
 
+func printTokenUsage(response openai.ChatCompletionResponse) {
+	fmt.Printf(
+		"PromptTokens:\t\t%d\nCompletionTokens:\t%d\nTotalTokens:\t\t%d\n",
+		response.Usage.PromptTokens,
+		response.Usage.CompletionTokens,
+		response.Usage.TotalTokens,
+	)
+}
+
+func (oac *OpenAiClient) getCompletion(messages []openai.ChatCompletionMessage) (string, error) {
+	resp, err := oac.client.CreateChatCompletion(
+		context.Background(),
+		openai.ChatCompletionRequest{
+			Model:    openai.GPT3Dot5Turbo,
+			Messages: messages,
+		},
+	)
+
+	if err != nil {
+		return "", err
+	}
+
+	printTokenUsage(resp)
+	return resp.Choices[0].Message.Content, nil
+}
+
 func (oac *OpenAiClient) GetCompletion(prompt string, debug bool) (*string, error) {
```

Comment:
Consider refactoring the `printTokenUsage` function into a separate utils file for better organization and reusability.